### PR TITLE
Edit mode

### PR DESCRIPTION
**Pull request by konradscheng**
https://app.asana.com/0/1199117989666292/1199132519920619/f

### CODE REVIEW

File: src/components/dashboard/menu-table/menu-table.js [hunk 1]
```diff
@@ -93,6 +74,76 @@ const MenuControls = styled.div`
...
+const DeleteButton = styled.div`
+  display: ${ props => (props.showEditMode ? "block" : "none")};
+  border: 2px solid #FA3838;
+  padding: 10px 20px;
+  background: #FA3838;
+  border-radius: 6px;
+  cursor: pointer;
+  width: 100px;
+`
+
```
Comment (cchyung) [inline src/components/dashboard/menu-table/menu-table.js:146]: Can we consolidate these button stylings?  Looks like these are all basically the same styling

**Review by cchyung — COMMENTED** [review]
Looks good, just small nit on the styling of buttons.

Also make sure to rebase on the develop branch so that your code is up to date with the latest dev branch.
`git checkout develop`
`git pull`
`git checkout <your branch>`
`git rebase -i develop`

File: src/components/dashboard/menu-table/menu-table.js (hunk 1 above)
Comment (konradscheng) [reply src/components/dashboard/menu-table/menu-table.js:147]: fixed

**Comment by cchyung** [comment]
@konradscheng see my slack messages re: rebase

**Review by cchyung — APPROVED** [review]

**Comment by cchyung** [comment]
gj!

**Pull request merged**